.gitignore

```diff
@@ -5,3 +5,4 @@
 /dist/
 __pycache__/
 liquidctl/_version.py
+.DS_Store
```

Comment:
I would suggest moving this to your system wide gitignore file (`~/.gitignore` or `~/.config/git/ignore`) because they are mac specific. 

To remove files from git run. 
`git rm .DS_Store liquidctl/.DS_Store liquidctl/driver/.DS_Store`

if the deletion is not acknowledged by git you can you `git add` those files after they are deleted.